bot/cogs/commands/useful.py

```diff
@@ -683,11 +683,10 @@ async def reminders_list(self, ctx: Ctx):
 
         for reminder in user_reminders:
             unix_timestamp = format_dt(reminder["at"], style="R")
-            reminder = reminder["reminder"]
-            if len(reminder) > 75:
-                reminder = shorten_text(reminder, 75)
+            if len(reminder["reminder"]) > 75:
+                reminder["reminder"] = shorten_text(reminder["reminder"], 75)
```

Comment:
<img width="466" alt="image" src="https://user-images.githubusercontent.com/38477514/219881232-7fe4ba4b-f5a3-4996-9d14-926ac0ea3581.png">
This function has that logic built in already.